alacritty.yml

```diff
@@ -231,6 +231,9 @@ hide_cursor_when_typing: false
 # - Beam
 cursor_style: Block
 
+# Whether the cursor should be a hollow block on window focus loss
+cursor_hollow_unfocused: true
```

Comment:
I'd prefer a name like `unfocused_hollow_cursor` over `cursor_hollow_unfocused` because it puts the condition under which the setting is relevant at the beginning. But maybe that's just preference.